src/main/java/frc/robot/Sensors.java

```diff
@@ -81,6 +95,11 @@ public CoralEnum getSensorState() {
       return simState;
     }
 
+    // If parallel sensor (sensor5) doesn't detect anything, there's no coral
+    if (!sensor5.getBeamBroke()) {
+      return CoralEnum.NO_CORAL;
+    }
+
```

Comment:
sensor5 detects alge, if its not triggered then there is not algae. There may still be coral